pjsip/include/pjsua2/endpoint.hpp

```diff
@@ -1696,6 +1838,33 @@ class Endpoint
     virtual void onMediaEvent(OnMediaEventParam &prm)
     { PJ_UNUSED_ARG(prm); }
 
+    /**
+     * Callback for computation of the digest credential.
```

Comment:
Perhaps a bit more info about when app needs to implement this callback, for example:
```Normally application does not need to implement this callback. Currently, this callback needs to be implemented only if application needs to support Digest AKA auth AND does not want to use the default digest computation backend (i.e: using libmilenage)```.

Reply:
Updated commit by an improved comment.